packages/static-renderer/src/pm/markdown/markdown.ts

```diff
@@ -78,7 +78,8 @@ export function renderToMarkdown({
             return `\n${serializeChildrenToHTMLString(children)}\n`
           }
 
-          return `\n${serializeChildrenToHTMLString(children[0])}| ${new Array(node.childCount - 2).fill('---').join(' | ')} |\n${serializeChildrenToHTMLString(children.slice(1))}\n`
+          const columnCount = node.children[0].childCount
+          return `\n${serializeChildrenToHTMLString(children[0])}| ${new Array(columnCount).fill('---').join(' | ')} |\n${serializeChildrenToHTMLString(children.slice(1))}\n`
```

Comment:
I didn't use `serializeChildrenToHTMLString` since a while, I guess it will add the newline afterwards so the insertion of the column `---` dividers is on the right new line?